.github/workflows/kurtosis_devnet.yaml

```diff
@@ -19,12 +19,6 @@ jobs:
           version: ${{ env.MISE_VERSION }}
           experimental: true
       - run: just run-kurtosis-devnet
-      - uses: Swatinem/rust-cache@v2 # cache for `just build-native`
-      # We build the client here because `just run-client-native-against-devnet`
-      # first polls op-geth for a first finalized block, which takes a few minutes,
-      # and only after that runs `cargo run` which then builds the client, and takes another
-      # few minutes. Better to build here while waiting for first l2 block to finalize.
-      - run: RISC0_SKIP_BUILD=1 SP1_SKIP_PROGRAM_BUILD=true just build-native
```

Comment:
why are you removing this? The failing CI you linked to just seems to be related to a crate name having changed?
![image](https://github.com/user-attachments/assets/996fd634-8cc3-4a8f-af4e-8d544fae56a5)